alzueta.yaml

```diff
@@ -0,0 +1,1849 @@
+  rate-constant: {A: 1.3e+14, b: 0.0, Ea: 4.6931e+04}
+- equation: H2NN + H <=> NNH + H2  # Reaction 258
+  rate-constant: {A: 4.8e+08, b: 1.5, Ea: -894.0}
+- equation: H2NN + H <=> tHNNH + H  # Reaction 259
+  rate-constant: {A: 7.0e+13, b: 0.0, Ea: 0.0}
+- equation: H2NN + O <=> NNH + OH  # Reaction 260
+  rate-constant: {A: 3.3e+08, b: 1.5, Ea: -894.0}
+- equation: H2NN + O <=> NH2 + NO  # Reaction 261
+  rate-constant: {A: 7.0e+13, b: 0.0, Ea: 0.0}
+- equation: H2NN + OH <=> NNH + H2O  # Reaction 262
+  rate-constant: {A: 2.4e+06, b: 2.0, Ea: -1192.0}
+- equation: H2NN + OH => NH2 + NO + H  # Reaction 263
+  rate-constant: {A: 2.0e+12, b: 0.0, Ea: 0.0}
+- equation: H2NN + HO2 => NH2 + NO + OH  # Reaction 264
+  rate-constant: {A: 9.0e+12, b: 0.0, Ea: 0.0}
+- equation: H2NN + HO2 <=> NNH + H2O2  # Reaction 265
+  rate-constant: {A: 2.9e+04, b: 2.69, Ea: -1600.0}
+- equation: H2NN + O2 <=> NH2 + NO2  # Reaction 266
+  rate-constant: {A: 1.5e+12, b: 0.0, Ea: 5961.0}
+- equation: H2NN + NH2 <=> NNH + NH3  # Reaction 267
+  rate-constant: {A: 1.8e+06, b: 1.94, Ea: -1152.0}
+- equation: HOCO + NO <=> CO2 + HNO  # Reaction 268
+  rate-constant: {A: 5.0e+09, b: 1.0, Ea: 0.0}
+- equation: HOCO + NH2 <=> CO2 + NH3  # Reaction 269
+  rate-constant: {A: 2.0e+13, b: 0.0, Ea: 0.0}
+- equation: H2NCO (+M) <=> CO + NH2 (+M)  # Reaction 270
+  type: falloff
+  low-P-rate-constant: {A: 1.0e+14, b: 0.0, Ea: 2.17e+04}
+  high-P-rate-constant: {A: 5.9e+12, b: 0.0, Ea: 2.5e+04}
+- equation: H2NCO + H <=> HNCO + H2  # Reaction 271
+  rate-constant: {A: 3.0e+13, b: 0.0, Ea: 0.0}
+- equation: H2NCO + O <=> HNCO + OH  # Reaction 272
+  rate-constant: {A: 3.0e+13, b: 0.0, Ea: 0.0}
+- equation: H2NCO + OH <=> HNCO + H2O  # Reaction 273
+  rate-constant: {A: 3.0e+13, b: 0.0, Ea: 0.0}
+- equation: HNCO + M <=> CO + NH + M  # Reaction 274
+  type: three-body
+  rate-constant: {A: 1.1e+16, b: 0.0, Ea: 8.6e+04}
+  efficiencies: {N2: 1.5}
+- equation: HNCO + H <=> NH2 + CO  # Reaction 275
+  rate-constant: {A: 3.6e+04, b: 2.49, Ea: 2345.0}
+- equation: HNCO + H <=> NCO + H2  # Reaction 276
+  rate-constant: {A: 9.0e+07, b: 1.66, Ea: 1.39e+04}
+- equation: HNCO + O <=> NCO + OH  # Reaction 277
+  rate-constant: {A: 2.2e+06, b: 2.11, Ea: 1.143e+04}
+- equation: HNCO + O <=> NH + CO2  # Reaction 278
+  rate-constant: {A: 9.6e+07, b: 1.41, Ea: 8520.0}
+- equation: HNCO + O <=> HNO + CO  # Reaction 279
+  rate-constant: {A: 1.5e+08, b: 1.57, Ea: 4.4012e+04}
+- equation: HNCO + OH <=> NCO + H2O  # Reaction 280
+  rate-constant: {A: 3.5e+07, b: 1.5, Ea: 3600.0}
+- equation: HNCO + HO2 <=> NCO + H2O2  # Reaction 281
+  rate-constant: {A: 3.0e+11, b: 0.0, Ea: 2.2e+04}
+- equation: HNCO + O2 <=> HNO + CO2  # Reaction 282
+  rate-constant: {A: 2.0e+13, b: 0.0, Ea: 5.89e+04}
+- equation: HNCO + H2O <=> NH3 + CO2  # Reaction 283
+  rate-constant: {A: 2.0e+13, b: 0.0, Ea: 4.85e+04}
+- equation: HNCO + NH <=> NH2 + NCO  # Reaction 284
+  rate-constant: {A: 3.0e+13, b: 0.0, Ea: 2.37e+04}
```

Comment:
As always, the last line of the file should end in a newline character. It's easy to check here in GitHub: if you see the ⛔ symbol at the end, your file is missing that newline.